app/browser/api/ledger.js

```diff
@@ -1408,6 +1408,31 @@ const clientprep = () => {
   _internal.verboseP = ledgerClient.prototype.boolion(process.env.LEDGER_PUBLISHER_VERBOSE)
 }
 
+const roundTripFromWindow = (params, callback) => {
```

Comment:
could you please add a jsdoc comment documenting the param options and the parameters that `callback` takes?

Reply:
see https://github.com/brave/browser-laptop/pull/13115/files#diff-2565914e73f517d565765ef02f30d306R102 for an example

Reply:
done